src/zap/zapimport.cpp

```diff
@@ -827,13 +827,15 @@ class ZapVirtualMethodThunk : public ZapImport
     virtual void Save(ZapWriter * pZapWriter);
 };
 
+#ifdef DEBUG
 #ifndef BINDER // Binder sees a body in vm\util.hpp
 static BOOL FitsInU2(unsigned __int64 val)
```

Comment:
There is FitsIn<...>() template in utilcode. It may be better to delete this function and replace the one use with the template instead of adding more cruft around it.